interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,430 @@ components:
+        This state must transition to Ended.  In this state, the
+        `/uss/v1/operational_intents/{entityid}/telemetry` USS-USS endpoint
+        should respond, if available, to queries from USS peers.  The create or
+        update request for this operational intent may omit a Key in this case
+        because the operational intent is being adjusted as flown and cannot
+        necessarily deconflict.
+      type: string
+      enum:
+      - Accepted
+      - Activated
+      - Nonconforming
+      - Contingent
+    Key:
+      description: >-
+        Proof that a client has obtained the latest airspace content, used to
+        ensure that write operations to the DSS occur only when the latest
+        content is known (i.e. has been read). The client is required to pass a
+        current Key constructed from information obtained with previous read
+        operations and subsequent requests for full information, and optionally,
+        subscription updates for deconflicted write operations to the DSS.  The
+        contents of this data structure are generated by the client.
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityOVN'
+      default: [ ]
+    SubscriptionUssBaseURL:
+      description: >-
+        The base URL of a USS implementation of the parts of the USS-USS API
+        necessary for receiving the notifications requested by this
+        subscription.
+      anyOf:
+        - $ref: "#/components/schemas/UssBaseURL"
+
+    ImplicitSubscriptionParameters:
+      description: >-
+        Information necessary to create a subscription to serve a single
+        operational intent's notification needs.
+      type: object
+      required:
+      - uss_base_url
+      properties:
+        uss_base_url:
+          description: >-
+            The base URL of a USS implementation of the parts of the USS-USS API
+            necessary for receiving the notifications that the operational
+            intent must be aware of.  This includes, at least, notifications for
+            relevant changes in operational intents.
+          anyOf:
+          - $ref: '#/components/schemas/SubscriptionUssBaseURL'
+        notify_for_constraints:
+          description: >-
+            True if this operational intent's subscription should trigger
+            notifications when constraints change. Otherwise, changes in
+            constraints should not trigger notifications.  The scope
+            utm.constraint_processing is required to set this flag true, and a
+            USS performing the constraint processing role should set this flag
+            true.
+          type: boolean
+          default: false
+          
+    #########
+    # The definitions before this seperator are from the standard UTM API (https://github.com/astm-utm/Protocol/blob/master/utm.yaml), the definitions specific for the automated testing suite follow below.
```

Comment:
This is not actually accurate because content above line 39 is not from this API -- I would recommend two big separators something like this:

```
################################################################################
#################### Start of ASTM-standard definitions    #####################
#################### interfaces/astm-utm/Protocol/utm.yaml #####################
################################################################################
```

then later

```
################################################################################
####################### End of ASTM-standard definitions #######################
################################################################################
```

Reply:
Thank you for this, I have now updated the two with your suggestions.